lib/src/main/kotlin/starknet/data/types/execution.kt

```diff
@@ -8,7 +8,7 @@ import starknet.data.types.Felt
 @Serializable
 data class Call(
     @SerialName("contract_address") val contractAddress: Felt,
-    val entrypoint: String,
+    val entrypoint: Felt,
```

Comment:
Why?

Reply:
Imo it's more aligned with other imputs, as we require Felt almost everywhere. And it eliminates situation when it's unclear whether someone should type selector name or the hex string of it. But it is also less developer friendly, so let's discuss which one is better